src/com/clover/remote/client/CloverConnectorBroadcaster.ts

```diff
@@ -178,8 +178,8 @@ export class CloverConnectorBroadcaster
         this.logger.debug('Sending Disconnect notification to listeners', message);
         this.listeners.forEach((listener: sdk.remotepay.ICloverConnectorListener) => {
             try {
-                listener.onDeviceDisconnected(message);  // changed the name in 1.3
-                listener.onDisconnected(message);        // left here for backwards compatibility.  Deprecated in 1.3*
+                listener.onDeviceDisconnected();  // changed the name in 1.3
```

Comment:
If "message" is no longer required, then it should also be removed from the logger.debug statement

Reply:
The interface never accepted a message.  Perhaps, it should?

Reply:
I would agree that it should perhaps accept a message, because we'd like to know how/why we disconnected if possible.